panther_driver/src/driver_node.py

```diff
@@ -1,323 +1,361 @@
 #!/usr/bin/python3
 
-import canopen
 import math
+import time
```

Comment:
If time is used only to sleep it should be `from time import sleep` like in other nodes

Reply:
+1